app/controllers/faculty_access_controller.rb

```diff
@@ -36,6 +37,12 @@ def pending
 
   protected
 
+  # def redirect_to_sheerid_if_newflow
+  #   if request.get? && Settings::Db.store.educator_feature_flag && !params[:bpff].present?
+  #     redirect_to(educator_sheerid_form)
+  #   end
+  # end
+
```

Comment:
Totally forgot about these comments in this controller. I'll delete them when I address any other suggestions you may have.